synapse_pay_rest.gemspec

```diff
@@ -1,17 +1,26 @@
+# coding: utf-8
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'synapse_pay_rest/version'
 
-Gem::Specification.new do |s|
-  s.name        = 'synapse_pay_rest'
-  s.version     = '0.0.11'
-  s.date        = %q{2016-07-08}
-  s.summary     = "SynapsePay v3 Rest API Wrapper"
-  s.description = "A simple ruby wrapper for the SynapsePay v3 Rest API"
-  s.authors     = ["Thomas Hipps", "Nowsta Team"]
-  s.email       = 'eric@nowsta.com'
-  s.require_paths = ["lib"]
-  s.files       = Dir.glob("{lib}/**/*")
-  s.homepage    = 'https://github.com/Nowsta/SynapsePayRest-Ruby'
-  s.license     = 'MIT'
-  s.add_dependency "rest-client"
+Gem::Specification.new do |spec|
+  spec.name        = 'synapse_pay_rest'
+  spec.version     = SynapsePayRest::VERSION
+  spec.authors     = ["Thomas Hipps", "Nowsta Team"]
```

Comment:
A brave and noble soul.... lol... I think he was the original author of the synapse client. At least he was listed as the author in the original codebase.